examples/dev/testManiFactors.jl

```diff
@@ -3,6 +3,24 @@ using IncrementalInference
 using StaticArrays
 using Manifolds
 
+
+##
+
+
+
+fg = initfg()
+
+v0 = addVariable!(fg, :x0, Pose2)
+v1 = addVariable!(fg, :x1, Point2)
+v2 = addVariable!(fg, :x2, Pose2)
+
+fg.solverParams.graphinit = false
+
+addFactor!(fg, [:x0], Prior(Normal()))
+addFactor!(fg, [:x0,:x1], Prior(Normal()))
+addFactor!(fg, [:x1,:x2], Prior(Normal()))
+addFactor!(fg, [:x0,:x2], Prior(Normal()))
```

Comment:
Regular use is `PriorPose2` `PriorPoint2` etc, don't worry about trying to make all work through just generic `Prior`

Reply:
That sneaked in somehow, File is just a sandbox that can become tests later.